en/guide/camera_definition_file.md

```diff
@@ -0,0 +1,65 @@
+# Camera Definition File
+
+As defined in the [MAVLink Camera Protocol](https://mavlink.io/en/protocol/camera.html), a [Camera Definition File](https://mavlink.io/en/protocol/camera_def.html) may be used to specify information about a camera's advanced settings and parameters. A camera will supply the URI for its definition file on request (if one has been defined), so that the file can be downloaded and parsed by the client.
+
+The *Camera Definition Files* can be hosted anywhere that is accessible to clients (including *QGroundControl* and Dronecode SDK apps). Often the files are served from the computer running CSD.
+
+CSD is able to supply the URIs for attached cameras using a mapping given in the [CSD Configuration File > \[uri section\]](../guide/configuration_file.md#uri).
+
+This topic explains how to:
+* Serve the definition files on the CSD companion computer.
+* Map attached cameras to their definition file URIs in the CSD configuration file.
+
+> **Tip** Camera definition files for a number of cameras are provided in [/samples/def](https://github.com/intel/camera-streaming-daemon/tree/master/samples/def).
+
+
+## Hosting Camera Definition Files
+
+Often the computer that is hosting CSD also serves the *Camera Definition Files*. If this is needed, we recommend using the Python *SimpleHTTPServer* because it is lightweight, and because the fact that it is present in many Linux distributions (as part of Python) means that it requires no additional setup.
+
+> **Note** Intel Aero uses *SimpleHTTPServer* by default (so this does not need to be set up on Aero).
+
+<span></span>
+> **Tip** The definition files may be hosted at any URI that is *accessible to requesting clients*. On some connected systems this could mean "anywhere on the Internet".
+
+To run *SimpleHTTPServer*:
+
+1. Open a terminal in the directory that you want to serve.
+2. Enter the following command to start the server on port 8080 (or specify another port)
+   ```
+   python -m SimpleHTTPServer 8080
```

Comment:
In the conf files, I have used port 8000. To keep the command to run http server default
`python -m SimpleHTTPServer`